Examples/Python/python/acts/examples/simulation.py

```diff
@@ -111,7 +111,7 @@ def addParticleGun(
                 multiplicity=FixedMultiplicityGenerator(n=multiplicity),
                 vertex=vtxGen
                 or acts.examples.GaussianVertexGenerator(
-                    mean=acts.Vector4(0, 0, 0, 0),
+                    mean=acts.Vector4(100, 0, 0, 0),
```

Comment:
We should probably keep the default as it was.

Reply:
I applied the changes in this commit : https://github.com/acts-project/acts/pull/3446/commits/9ec780a31502d0253d6cb72f9697f4da258159b0